Documentation/project-docs/oss-signing.md

```diff
@@ -1,18 +0,0 @@
-Open Source Signing
```

Comment:
I think it'd make sense to keep this document for a while to not break links and explain that OSS Signing was the old term and link to the public signing doc.